README.md

```diff
@@ -975,18 +974,31 @@ def test_backup_action():
 
     # If you didn't declare do_backup in the charm's metadata, 
     # the `ConsistencyChecker` will slap you on the wrist and refuse to proceed.
-    out: scenario.ActionOutput = ctx.run_action(ctx.on.action("do_backup"), scenario.State())
+    state = ctx.run(ctx.on.action("do_backup"), scenario.State())
 
-    # You can assert action results, logs, failure using the ActionOutput interface:
-    assert out.logs == ['baz', 'qux']
-    
-    if out.success:
-      # If the action did not fail, we can read the results:
-      assert out.results == {'foo': 'bar'}
+    # You can assert action results and logs using the action history:
+    assert ctx.action_output.logs == ['baz', 'qux']
+    assert ctx.action_output.results == {'foo': 'bar'}
```

Comment:
Seeing these here makes we wonder if it should just be `ctx.action_logs` and `ctx.action_results`. Flat is better than nested, and most everything else here we've flattened. Likely most of the time tests will only be asserting on `ctx.action_results` anyway.

That would also solve the "ops vs scenario ActionOutput" question, because there wouldn't be an `ActionOutput` object anymore.